paasc/src/lib.rs

```diff
@@ -0,0 +1,53 @@
+use rustls::{ciphersuite, internal::pemfile, ClientConfig, RootCertStore, SupportedCipherSuite};
+use std::{fs::File, io::BufReader};
+use tonic::transport::Channel;
+
+use paas_types::process_service_client::ProcessServiceClient;
+
+static CIPHERSUITES: &[&SupportedCipherSuite; 5] = &[
+    &ciphersuite::TLS13_AES_256_GCM_SHA384,
+    &ciphersuite::TLS13_CHACHA20_POLY1305_SHA256,
+    &ciphersuite::TLS13_AES_128_GCM_SHA256,
+    &ciphersuite::TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
+    &ciphersuite::TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+];
+
+fn buf_read(path: &str) -> Result<BufReader<File>, Box<dyn std::error::Error>> {
+    Ok(BufReader::new(File::open(path)?))
+}
+
+fn rustls_config(client: &str) -> Result<ClientConfig, Box<dyn std::error::Error>> {
+    let mut cert_store = RootCertStore::empty();
+    cert_store
+        .add_pem_file(&mut buf_read("data/server_ca.pem")?)
+        .unwrap();
+
+    let cert_path = format!("data/{}.pem", client);
```

Comment:
Consider using `std::path` facilities here.

Reply:
Will fix in client PR (WIP branch) in order to minimize conflicts. Hope that is fine.